### DISCUSSION TITLE
Why?

### DISCUSSION THREAD

**Discussion by koyuawsmbrtn** [microsoft/sudo#49] [Q&A] [answered]
Why though?

**Comment by FinnT730** [comment]
Why not?
Why do extra clicks, then you can just do winkey -> type cmd -> sudo {command here}
No need for mouse

**Reply by MrBreakNFix** [reply]
@FinnT730 
win + R
cmd
ctrl + shift + enter

**Comment by DHowett** [comment, marked as answer]
This one has been answered sufficiently. You are welcome to ignore sudo if it seems redundant or does not fit your use case.
